sig-scheduling/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to SIG Scheduling
+
+Welcome to contributing to SIG Scheduling. We are excited about the prospect of you
+joining our [community](https://github.com/kubernetes/community/tree/master/sig-scheduling)!
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+You can read the SIG mission outlined in the [charter](https://git.k8s.io/community/sig-cluster-lifecycle/charter.md).
+
+We have multi areas you can contribute to, PRs, issues, documentations, new ideas,
+helping to answer end-user's questions, attending meetings, all kinds of contributions are welcomed.
+
+## Before you begin
+
+We strongly recommend you to check out the [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide)
+and [Contributor Cheat Sheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
+
+## Getting Started
+
+* If you're a newcomer and have no idea where to start, we have a non-stale pool of issues that are
+available for you:
+  * [first-good-issue](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Asig%2Fscheduling+)
+  * [help wanted](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+label%3Asig%2Fscheduling+)
+
+* If you want to understand the architectural details of scheduler, you can refer to the [design docs](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-scheduling)
+  or [KEPs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling).
+
+* If you have any questions, you can find us in the [slack](https://app.slack.com/client/T09NY5SBT/C09TP78DV).
+
+* If you find a bug, please open an issue under kubernetes/kubernetes with labels `/sig scheduling` and `/kind bug`,
+also follow the requirements of `Bug Report`, it will help us a lot when analyzing the errors.
+
+* If you have a feature request:
+  * Open an issue under kubernetes/kubernetes with labels `/sig scheduling` and `/kind feature`, please focus on
+  the user stories
+  * If there's any debate on the rationalities, you can bring it to the [SIG meeting](https://github.com/kubernetes/community/tree/master/sig-scheduling#meetings).
+  * If there're multiple implementation options, consider to create a doc with Pros and Cons, and gather some feedbacks.
+  * If necessary, open an issue in kubernetes/enhancements and write a KEP for it.
+
+* If you find any out-of-date documentations, please help us to revise that, the readers come latter
```

Comment:
```suggestion
* If you find any out-of-date documentation, please help us to revise that. Future readers
```

Maybe we can explain to open issues in k/website for end-user documentation and in this k/community for developer docs

Reply:
Make sense, that will be really friendly to readers.